### PR TITLE
Added Laravel 6.0.x compatibility

### LINKED ISSUE #20: Add support for Laravel 6.0
Can you add a support for Laravel 6.0?

### PR DESCRIPTION
**Pull request by janmaarten**
## Description

Thix fixes #20 in the v2 branch, and fixes a code style issue.

## Motivation and context

I'd like to use the package with Laravel 6.0

## How has this been tested?

I've tested this with a clean Laravel 6 installation.

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

Go over all the following points, and put an `x` in all the boxes that apply.

Please, please, please, don't send your pull request until all of the boxes are ticked. Once your pull request is created, it will trigger a build on our [continuous integration](http://www.phptherightway.com/#continuous-integration) server to make sure your [tests and code style pass](https://help.github.com/articles/about-required-status-checks/).

- [x] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [x] My pull request addresses exactly one patch/feature.
- [x] I have created a branch for this patch/feature.
- [x] Each individual commit in the pull request is meaningful.
- [x] I have added tests to cover my changes.
- [x] If my change requires a change to the documentation, I have updated it accordingly.

If you're unsure about any of these, don't hesitate to ask. We're here to help!

### CODE REVIEW

**Comment by janmaarten** [comment]
Hi @sandervanhooft, this is my first PR _ever_, so please do let me know if I should things otherwise. Thanks!

File: tests/CreateTestModelsTable.php [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <?php
 
+namespace SanderVanHooft\Invoicable;
```
Comment (sandervanhooft) [inline tests/CreateTestModelsTable.php]: Can you try reverting this? Not sure, but it seems to trip up the Travis tests.

**Comment by sandervanhooft** [comment]
Thanks! One tip: keep an eye on the Travis build status.

File: tests/CreateTestModelsTable.php (hunk 1 above)
Comment (janmaarten) [reply tests/CreateTestModelsTable.php]: Yes, this trips Travis up, but if I simply remove this, the style check fails.

**Comment by sandervanhooft** [comment]
Nice first PR @janmaarten, thanks! 🎉

**Comment by sandervanhooft** [comment]
Just released it (`1.1.0`)

**Comment by janmaarten** [comment]
Ha, so proud 💃  Thanks :)

**Comment by janmaarten** [comment]
Hm, it seems like you tagged the master branch from 05/12/2018 as 1.1.0 @sandervanhooft.

**Comment by sandervanhooft** [comment]
Ouch, seems like I missed that you were working on the v2 branch, which is still in development

**Comment by sandervanhooft** [comment]
#22

**Comment by sandervanhooft** [comment]
My bad. Just released `1.2.0` to fix this.

https://github.com/sandervanhooft/laravel-invoicable/releases/tag/1.2.0

**Pull request merged**